components/ActivityIndicator/examples/ActivityIndicatorExample.m

```diff
@@ -17,10 +17,10 @@
 #import <UIKit/UIKit.h>
 
 #import "ActivityIndicatorExampleSupplemental.h"
+#import "CatalogStyle.h"
```

Comment:
It took me days in my prior PR to track down every single color definition in the catalog and make them all consistent. By creating a central place where the color palette of the catalog is defined, it will make the current catalog and future demos in the catalog more consistent and prevent the extra work.

Are there any alternatives you can think of that can help bring consistency?